cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -2375,6 +2375,7 @@ impl_runtime_apis! {
 				code,
 				data,
 				salt,
+				NonceAlreadyIncremented::Yes,
```

Comment:
This is a dry-run. Only transactions do increment.

```suggestion
				NonceAlreadyIncremented::Yes,
```